### PR TITLE
Setup Sphinx Docs

### PR DESCRIPTION
**Pull request by Zsailer**
Setting up an initial Sphinx directory with a landing page and some documentation on JSON schema.

### CODE REVIEW

**Comment by Zsailer** [comment]
Any objections to merging this? This just adds sphinx docs to the repo. After we merge this, I'll create a readthedocs page for this project.

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+=================
+
+**Configurable event-logging for Jupyter applications and extensions.**
+
+
+Telemetry provides a configurable traitlets object, EventLog, for structured event-logging in Python. It leverages Python's standard logging library for filtering, handling, and recording events. All events are validated (using jsonschema) against registered JSON schemas.
+
+If you're looking for telemetry in Jupyter frontend applications (like JupyterLab), checkout the work happening in jupyterlab-telemetry_! 
+
+.. _jupyterlab-telemetry: https://github.com/jupyterlab/jupyterlab-telemetry
```
Comment (Zsailer) [inline docs/index.rst]: @jaipreet-s Great idea. What do you think of this line on the front page of the docs?

Comment (jaipreet-s) [reply docs/index.rst]: SGTM!

**Comment by Zsailer** [comment]
Thanks, @jaipreet-s. Merging!

**Pull request merged**